standards/SPIFFE_Federation.md

```diff
@@ -0,0 +1,316 @@
+5.2.2.1. [Endpoint URL Requirements](#5221-endpoint-url-requirements)  
+5.2.2.2. [Endpoint Parameters](#5222-endpoint-parameters)  
+5.2.2.3. [Serving a Bundle Endpoint](#5223-serving-a-bundle-endpoint)  
+5.2.2.4. [Consuming a Bundle Endpoint](#5224-consuming-a-bundle-endpoint)  
+6\. [Relationship Lifecycle](#6-relationship-lifecycle)  
+6.1. [Establishing a Relationship](#61-establishing-a-relationship)  
+6.2. [Maintaining a Relationship](#62-maintaining-a-relationship)  
+6.3. [Terminating a Relationship](#63-terminating-a-relationship)  
+6.4. [Lifecycle Diagram](#64-lifecycle-diagram)  
+7\. [Security Considerations](#7-security-considerations)  
+7.1. [Distribution of Endpoint Parameters](#71-distribution-of-endpoint-parameters)  
+7.2. [Explicitly-defined Endpoint Parameters](#72-explicitly-defined-endpoint-parameters)  
+7.3. [Preserving the `<Trust Domain, Bundle>` Binding](#73-preserving-the-trust-domain-bundle-binding)  
+7.4. [Trustworthiness of the Bundle Endpoint Server](#74-trustworthiness-of-the-bundle-endpoint-server)  
+7.5. [Authenticity of the Bundle Endpoint](#75-authenticity-of-the-bundle-endpoint)  
+7.5.1. [Bundle Endpoint URL Redirection](#751-bundle-endpoint-url-redirection)  
+7.5.2. [Network Traffic Interception](#752-network-traffic-interception)  
+7.5.3. [Endpoint Parameters](#753-endpoint-parameters)  
+7.6. [Chaining Trust with `https_spiffe`](#76-chaining-trust-with-https_spiffe)  
+
+## 1. Background
+The SPIFFE specifications define the documents and interfaces necessary for establishing a platform-agnostic workload identity framework that is capable of bridging systems in different domains without the need for implementing identity translation or credential exchange logic. They define a “[trust domain][1]”, which serves as an identity namespace.
+
+SPIFFE is decentralized by nature. Each trust domain acts in its own capacity, under its own authority, and is administratively isolated from systems residing in other trust domains. Although trust domains delineate administrative and/or security domains, a core SPIFFE use case is enabling communication across these same boundaries where needed. Therefore, it is necessary to define a mechanism by which an entity may be introduced to a foreign trust domain, allowing it to authenticate identities issued by “other” SPIFFE authorities and allowing workloads in one trust domain to securely authenticate workloads in a foreign trust domain.
+
+A [SPIFFE bundle][2] is a self-contained resource that contains the public key material needed to authenticate identities in a particular trust domain. This document introduces a specification by which SPIFFE bundles can be securely fetched for the purpose of authenticating identities issued by external authorities. Included is information on how to serve a SPIFFE bundle, how to retrieve SPIFFE bundle, and how to authenticate the endpoints that serve them.
+
+## 2. Introduction
```

Comment:
This first paragraph has the first instance of the phrase "SPIFFE federation" in this doc. Is it worth defining exactly what this term means? Namely, calling out that "SPIFFE federation is the process of (cryptographically) authenticating to a peer in a different trust domain". 

I'm sure that this is spelled out in other SPIFFE docs, but it may be good to emphasize since "federation" can often be an ambiguous term.

Reply:
I feel like our use of "federation" is fairly disambiguated by the Background section.

Reply:
It does seem, to me, beneficial to have a "SPIFFE Federation is ..." statement right around here. I agree that we have clearly articulated the picture, but a blunt statement could never hurt?

The suggestion tool doesn't work with this comment, here is what I'd propose:
```
## 2. Introduction
SPIFFE Federation is the act of obtaining the materials necessary to validate SPIFFE identities in a different trust domain, and providing said materials to the workloads performing the authentication. ...
```

Reply:
"SPIFFE Federation is the act of obtaining the materials necessary to authenticate identity credentials (SVIDs) issued by a different trust domain, and providing said materials to the workloads performing the authentication."

I like "authenticate credentials" better than "validate identities", which might be confused with things like validating a purported SPIFFE ID is valid according to our new rules about character set.

Reply:
From SIG-Spec call: we want both the concept and the actual definition captured here. Something like below

SPIFFE Federation enables cross-trust domain authentication of identity credentials (SVIDs). Specifically, it is the act of obtaining ... a bundle